lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1,15 +1,35 @@
 {
+  "fetchTime": "2020-02-29T11:11:07.724Z",
```

Comment:
when updating the artifacts @jayaddison we try to keep the changes as minimal as possible, i.e. in this case *only* update the a11y artifact

```
yarn update:sample-artifacts Accessibility
```

Reply:
Gotcha, thanks!  For some reason I find the artifact updates tricky to manage.  I'll aim to take more time to read the related docs and code.